WordPress/WordPressTest/BlockEditorSettingsServiceTests.swift

```diff
@@ -233,8 +233,6 @@ class BlockEditorSettingsServiceTests: XCTestCase {
     }
 
     func testFetchBlockEditorSettings_OrgSite_NoPlugin() {
-        blog = BlogBuilder(context).build()
```

Comment:
The missing `wordPressVersion` was causing the failure. Decided to remove the new object creation since we don't customise it.